game/theater/seasonalconditions.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+from dataclasses import dataclass
+from enum import Enum
+from typing import Dict
```

Comment:
FYI, can just use `dict` since we're 3.9+ now if you prefer. No real difference.

Reply:
Wasn't aware! I'm no pythonista, just imitating other code ;) Will fix.